app/Resources/views/modules/header.html.twig

```diff
@@ -0,0 +1,45 @@
+<header>
+    <nav class="navbar navbar-default navbar-fixed-top">
+        <div class="container-fluid">
+            <div class="navbar-header">
+                <a class="navbar-brand" href="#"><img class="navbar-img__hidden" alt="Brand" src="{{ asset('img/logo-home.png') }}"></a>
+                <button type="button" class="hamburger navbar-toggle collapsed " data-toggle="collapse" data-target="#navbar" aria-expanded="false">
+                    <span class="sr-only navbar-iconBar">Toggle navigation</span>
+                    <span class="icon-bar navbar-iconBar"></span>
+                    <span class="icon-bar navbar-iconBar"></span>
+                    <span class="icon-bar navbar-iconBar"></span>
+                </button>
+            </div>
+            <div id="navbar" class="navbar-collapse collapse">
+                <ul class="nav navbar-nav">
+                    <li class="navbar-menu"><a class="navbar-link" href="">NOS RESTOS</a></li>
+                    <li class="navbar-menu"><a class="navbar-link" href="">RÉSERVER</a></li>
+                    <li class="navbar-menu"><a class="navbar-link" href="">BONS CADEAUX</a></li>
+                    <li class="navbar-menu" id="brand-logo"><a class="navbar-link" href=""><img class="navbar-img" src="{{ asset('img/logo-home.png') }}"></a></li>
+                    <li class="navbar-menu"><a class="navbar-link" href="">BLOG</a> </li>
+                    <li class="navbar-menu"><a class="navbar-link" href="">NOUS CONTACTER</a></li>
+                    {% if app.user %}  {# when user is connected , his firstname appears instead of connection link  #}
+                    <li class="dropdown myAccount">
+                        <a class="dropdown-toggle text-uppercase" data-toggle="dropdown" aria-expanded="false" >
```

Comment:
I believe we need a picture here, not sure if the functionality is ready, maybe add an issue ?